src/AddressUkPafLookupViewBridge.js

```diff
@@ -0,0 +1,139 @@
+var bridge = function (presenterPath) {
+    window.rhubarb.viewBridgeClasses.HtmlViewBridge.apply(this, arguments);
+};
+
+bridge.prototype = new window.rhubarb.viewBridgeClasses.HtmlViewBridge();
+bridge.prototype.constructor = bridge;
+
+bridge.prototype.attachEvents = function() {
+
+    var self = this,
+        alertClass = "c-alert",
+        manualAddressElements = $(".manual-fields"),
```

Comment:
This ViewBridge relies heavily upon jQuery, but the View doesn't require jQuery to be loaded, nor does the composer file list it as a dependency. That said, everything jQuery is used for here could easily be done without jQuery, and it may be useful not to have jQuery as a dependency if we intend to eventually release this as an SDK to support a public GCD PAF service. @acuthbert care to weigh in on that?